components/ArrowButton/ArrowButton.tsx

```diff
@@ -0,0 +1,41 @@
+"use client";
```

Comment:
Podemos sacar el `'use client'` para dejar este componente como un Server Component, ya que no se esta utilizando ningun tipo de client hook (como `useState` o `useEffect`)

- [Mas info aca](https://beta.nextjs.org/docs/rendering/server-and-client-components#convention)